demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -16,7 +16,7 @@
 
 public class MainActivity extends Activity {
 
-    private static final String URI_VIDEO_WIDEVINE_EXAMPLE_MODULAR_MPD = "https://storage.googleapis.com/wvmedia/cenc/h264/tears/tears.mpd";
```

Comment:
This asset wasn't very useful, it didn't have any audio tracks 😬 we need some custom content or at least an asset selection to make testing easier in the demo.

Reply:
👍 we should definitely create our own samples